CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Master
 
 * Cached asset file should never take precedent over existing file (#402)
+* Do not write `database.yml` when using active record >= 4.1 (previously we only detected >= Rails 4.1) (#403)
```

Comment:
Versions are correct. We switched from checking "rails" to checking "active record"